pipeline/assets/traffic_incidents.py

```diff
@@ -45,7 +47,18 @@ def roadworks_svzbw_datex2() -> None:
     """
     # Download and republish, if changed
     destination_folder = os.path.join(WEB_ROOT, *ROADWORKS_ASSET_KEY_PREFIX)
-    download(ROADWORKS_DATEX2_DOWNLOAD_URL, destination_folder, ROADWORKS_DATEXII_FIILENAME, create_precompressed=True)
+    auth = (
```

Comment:
One Prio B thing: normal if / else instead of inline and then make it multiline due line length is better readable.